composer.json

```diff
@@ -16,6 +16,7 @@
     "drupal/core": "^8.8",
     "drupal/components": "~1.0",
     "drupal/ds": "~3.3",
-    "drupal/ui_patterns": "~1.0"
+    "drupal/ui_patterns": "~1.0",
+    "drupal/twig_extender": "~2.4"
```

Comment:
I'm rethinking adding this contrib module. It saves me from writing a bunch of code but introduces another dependency that without, breaks anything that uses the layouts.

Reply:
But then again, we are dependant on a lot of things already.

Reply:
we actually don't really need `twig_extender`. All that really does is make adding new functions a plugin vs a service. I'd suggest going the way drupal core uses
https://medium.com/@thihathit/extending-custom-twig-extension-to-drupal-8-twig-extension-class-d4b99c2177ae

Reply:
It might not be bad to get `twig_tweaks` though. :man_shrugging:

Reply:
`twig_extender` sure makes writing the plugin easy tho. I was on the fence with it anyhow so I'll do it the good ol' fashioned way. I do like twig_tweaks but nothing in there we need at the moment.

Reply:
Does it though? its really almost the same amount of effort both ways. :man_shrugging: